src/components/Card.js

```diff
@@ -21,15 +43,47 @@ export default class Card {
         })
     }
 
-    _handleLikeClick (){
-        this._likeButton.classList.toggle('active');
+    escPress(evt) {
+        if (evt.key === 'Escape' && this._modalDelete) {
+            this._modalDelete.classList.remove('modal_opened');
+        }
+    }
+
+    _handleDeleteSubmit(evt) {
+        console.log('this is working');
+        evt.preventDefault(); // Prevent default delete submission
+        const messageElement = document.querySelector('#delete-button');
+        messageElement.textContent = 'Saving...';
+        setTimeout(() => this._modal.classList.remove('modal_opened'), 6000);
+        this._handleDeleteButton(this);
+        this._modalDelete.classList.remove('modal_opened'); // Close the modal
+        this._deleteSubmit.removeEventListener('submit', this._deleteSubmitHandler);
```

Comment:
Please, delete it

Every modal should be created using `Popup`. And submit handler should be added inside the class